browser/ui/views/toolbar/brave_toolbar_view.cc

```diff
@@ -191,6 +196,16 @@ void BraveToolbarView::OnEditBookmarksEnabledChanged() {
   Update(nullptr);
 }
 
+void BraveToolbarView::OnShowBookmarksButtonChanged() {
+  if (!bookmark_)
+    return;
+
+  DCHECK_EQ(DisplayMode::NORMAL, display_mode_);
+  bookmark_->SetVisible(browser_defaults::bookmarks_enabled &&
```

Comment:
How about having new method for this something like `UpdateBookmarkVisibility()` because 
`OnShowBookmarksButtonChanged()` and `Update()` methods have duplicated code?

Reply:
👍